docs/manual/src/Overview.md

```diff
@@ -0,0 +1,18 @@
+# uniffi
+
+Uniffi is a tool that automatically generates foreign-language bindings targeting Rust libraries.  
+It fits in the practice of consolidating business logic in a single Rust library while targeting multiple platforms, making it simpler to develop and maintain a cross-platform codebase.  
+Note that this tool will not help you ship a Rust library to these platforms, but simply not have to write bindings code by hand [[0]](https://i.kym-cdn.com/photos/images/newsfeed/000/572/078/d6d.jpg).
+
+## Design
+
+uniffi requires to write an Interface Definition Language ([IDL](https://heycam.github.io/webidl/)) file describing the methods and data structures available to the targeted languages.  
+This .idl file, whose definitions must match with the exposed Rust code, is then used to generate Rust *scaffolding* code and foreign-languages *bindings*. This process can take place either during the build process or be manually initiated by the developer.
```

Comment:
This page would really benefit from a high-level diagram, along the lines of the one we have in the README for application-services (but with a few extra bits for where uniffi fits in).